template/tsconfig.json

```diff
@@ -29,5 +29,5 @@
   },
   "exclude": ["node_modules"],
   "include": ["src", "test", "storybook"],
-  "files": [".d.ts"]
+  "files": ["template/.d.ts"]
```

Comment:
feels like we should revert this? In the consuming app there wont be a template directory so this would have to be updated.

Reply:
this was not intentional - may be weird merge error

Reply:
fixed!